wondrous-app/components/Common/MilestoneTask/MilestoneTasks.tsx

```diff
@@ -39,7 +44,7 @@ const MilestoneEmpty = styled.div`
   display: flex;
   align-items: center;
   justify-content: center;
-  background-color: #171717;
+  /* background-color: #171717; */
```

Comment:
can we remove this line if we don't need it?

Reply:
sure, actually I'll just remove this component since it's not being used anymore.

Reply:
done